notebooks/03_evaluate/evaluation_spark.ipynb

```diff
@@ -0,0 +1,136 @@
+{
+    "cells": [{
+        "cell_type": "markdown",
+        "source": ["# Spark Evaluation"],
+        "metadata": {}
+    }, {
+        "cell_type": "markdown",
+        "source": ["This notebook uses MMLSpark, please see github for installation details for your platform.\nhttps://github.com/Azure/mmlspark"],
```

Comment:
The code included in the first cell is "supposed" to install it into a local env for you, but this would not work on Databricks. I'm also very curious to see how the build server handles it...

`SUBMIT_ARGS = "--packages Azure:mmlspark:0.15 pyspark-shell"
os.environ["PYSPARK_SUBMIT_ARGS"] = SUBMIT_ARGS

from pyspark.sql import SparkSession

spark = (
   SparkSession.builder.appName("Spark Evaluation Metrics")
   .master("local[*]")
   .config("memory", "1G")
   .getOrCreate()
)`

Reply:
I was also thinking... the helper method we have... `get_or_create_spark()`, within it goes and starts Spark with the .getOrCreate() method call. Because this gets called within the helper, I can't further customize it... 
I am thinking it would make sense to refactor the method, and pull .getOrCreate() out into the actual notebooks...